### ISSUE TITLE
Only play audio stream, and automatically pause after 13 minutes

### ISSUE THREAD

**Issue by Unkn0wn0802** [samirkumardas/jmuxer#119]
audio buffer quota full
jmuxer.min.js:1 JMuxer cleaning audio buffer due to QuotaExceeded error

**Comment by Covfefeable** [comment]
same problem here, any solutions?

**Comment by samirkumardas** [comment]
@Covfefeable did you try setting `clearBuffer` to true?

**Comment by Covfefeable** [comment]
> @Covfefeable did you try setting `clearBuffer` to true?
yes, here is my config, I receive audio data from websocket, and feed realtime
`
      mode: 'audio',
      flushingTime: 20,
      debug: true,
      clearBuffer: true,
`
every time when it play for more than 25 minutes, there is no sound although websocket data is continue sending, console outputs buffer full xxxx, and clear doesn't work

**Comment by Covfefeable** [comment]
![error](https://github.com/user-attachments/assets/3430d37f-c607-4151-8634-9b8db9644e7f)
here is the log
